spec/features/delete_draft_collection_spec.rb

```diff
@@ -5,29 +5,38 @@
 
 RSpec.describe 'Delete a draft collection', js: true do
   let(:user) { create(:user) }
+  let(:collection_attrs) { attributes_for(:collection) }
+  let(:collection) { Collection.last }
 
   before do
     sign_in user, groups: ['dlss:hydrus-app-collection-creators']
+    visit dashboard_path
```

Comment:
Can we just `create(:collection)` here instead of going through the UI?  Presumably create a collection is tested elsewhere.

Reply:
I originally had used the factory when I first create this test in a previous PR, but it left the collection in a state where it couldn't be accessed by the user logging in (and therefore the delete button was not able to be rendered).  After trying various things and pairing with Naomi, we got the test to work by just creating it via the UI.  Maybe you have some ideas on how to get this work?